app/scripts/lib/createRPCMethodTrackingMiddleware.js

```diff
@@ -252,6 +255,18 @@ export default function createRPCMethodTrackingMiddleware({
         }
         ///: END:ONLY_INCLUDE_IF
 
+        const isRedesignedConfirmation =
+          process.env.ENABLE_CONFIRMATION_REDESIGN &&
+          preferencesController.store.getState().preferences
+            .redesignedConfirmations &&
+          ConfirmationRedesignTypes.includes(method);
+
+        if (isRedesignedConfirmation) {
+          eventProperties.ui_customizations = (
```

Comment:
Minor, but would it condense the syntax a little to use a spread operator such as:

```
eventProperties.ui_customizations = [
    ...eventProperties.ui_customizations,
    MetaMetricsEventUiCustomization.RedesignedConfirmation
];
```

Reply:
Thanks for bringing this up. The reason I haven't done this is I'm unsure how our data platform implementation handles empty arrays and we wouldn't want to add empty arrays to them. I can look into this more

Reply:
sgtm! updated to
```
eventProperties.ui_customizations = [
  ...(eventProperties.ui_customizations || []),
  MetaMetricsEventUiCustomization.RedesignedConfirmation,
];
```